lambda/docker-runner.yaml

```diff
@@ -0,0 +1,111 @@
+# This template depends on having an AWS EFS in place which can be deployed with
+# https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/EFS/efs.yaml
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: S3Objects
+Description: >-
+  Docker runner product
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: Container Image
+        Parameters:
+          - ImageUri
+          - EntryPoint
+          - Command
+          - WorkingDirectory
+          - MemorySize
+      - Label:
+          default: Synapse
+        Parameters:
+          - SynapseUser
+          - SynapsePassword
+      - Label:
+          default: Execution Trigger
+        Parameters:
+          - EnableSchedule
+          - Schedule
+Parameters:
+  MemorySize:
+    Type: Number
+    MinValue: 128
+    MaxValue: 10240
+    Default: 128
+    Description: >
+      The amount of memory available to the function at runtime. Increasing the function's
+      memory also increases its CPU allocation.
+  ImageUri:
+    Type: String
+    Description: >
+      The URI of the Amazon ECR) repository for the Lambda function's container image.
+      Example: 111111111.dkr.ecr.us-east-1.amazonaws.com/MY-DOCKER:latest
+  EntryPoint:
+    Type: String
+    Default: ""
+    Description: (Optional) The entry point to the application in the docker container
+  Command:
+    Type: CommaDelimitedList
+    Default: ""
+    Description: (Optional) Command separated list of parameters to pass in with docker ENTRYPOINT
+  WorkingDirectory:
+    Type: String
+    Default: ""
+    Description: (Optional) The working directory for the docker container
+  EnableSchedule:
+    Description: >
+      true to run on a schedule, false to disable. If enabled a valid Schedule must be provided
+    Type: String
+    Default: false
+    AllowedValues:
+      - true
+      - false
+  Schedule:
+    Description: >
+      Schedule to execute the docker, can be a rate or a cron schedule. Format at
+      https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html
+    Type: String
+    Default: cron(0 09 ? * MON *)  # Run every Monday at 9am
```

Comment:
This default should probably be run once or we may get a bunch of jobs running on Monday at 9 am repeatedly on accident

Reply:
The schedule is only enabled if the `EnableSchedule` parameter is true.   Please suggest something if you don't like the default.

Reply:
I suppose if the `EnableScheduled` parameter is false by default, it's not an issue. This is in the local TZ of the region, right?

Reply:
lambda is on UTC time. https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html
```
TZ – The environment's time zone (UTC). The execution environment uses NTP to synchronize the system clock.
```